### PR TITLE
feature/spendingscounter

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
With this PR I want to add a spendings counter to the category pages, displaying the total amount of spendings for the corresponding category.
in addition, I've adjusted some stylings and added secondary links (from the home page to the add spendings page and from the category pages back to the home page).
Thanks a lot for reviewing this

### CODE REVIEW

File: components/SpendingsCounter.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export default function SpendingsCounter({spendingInput, categoryName}) {
+  const allSpendingsValues = spendingInput
+    .filter(spendingInput => spendingInput.category === categoryName)
+    .reduce((allSpendingsValues, currentSpending) => {
+      return allSpendingsValues + Number.parseFloat(currentSpending.amount);
+    }, 0);
+
+  const totalSpendingsValue = allSpendingsValues;
```
Comment (stefankorder) [inline components/SpendingsCounter.js:10]: Why you don't use allSpendingsValues directly where you use totalSpendingsValue?

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good! Take a look at my comment!

File: components/CategoryOverview.js [hunk 2]
```diff
@@ -10,22 +12,45 @@ export default function CategoryOverview({spendingInput, categoryName}) {
...
+                            Amount: {spendingInput.amount} €
+                          </StyledLogCardListItem>
+                          <StyledLogCardListItem>
+                            Date: {spendingInput.date}
+                          </StyledLogCardListItem>
+                          <StyledLogCardListItem>
+                            Category: {spendingInput.category}
+                          </StyledLogCardListItem>
+                        </>
+                      }
```
[the remark this answers is not in the document]
Comment (marcos-de-barros-vioque) [reply components/CategoryOverview.js:43]: Ok, I see now, it's superfluous. Thanks, @stefankorder !

File: components/SpendingsCounter.js (hunk 1 above)
Comment (marcos-de-barros-vioque) [reply components/SpendingsCounter.js:10]: Ok, I see now,` totalSpendingsValue` it's superfluous. I'm adjusting it.

**Review by actyralf — APPROVED** [review]
Well done!

**Pull request merged**